exercises/beer-song/BeerSongExample.swift

```diff
@@ -0,0 +1,54 @@
+//
```

Comment:
Please remove this section from both files, so it matches the format of our other examples.